### DISCUSSION TITLE
Compatibility with node-red?

### DISCUSSION THREAD

**Discussion by gac9415** [EPMatt/awesome-ha-blueprints#265] [Q&A]
First off, your blueprints are wonderful and greatly appreciated! 

I set up an automation using a controller blueprint(IKEA E1812 TRÅDFRI Shortcut button). I have another trigger that I wanted to perform the same automation so I figured I'd use a call service to trigger the automation, but this apparently doesn't work?

View inside Node-Red
<img width="435" alt="Screen Shot 2022-02-20 at 2 39 35 AM" src="https://user-images.githubusercontent.com/28831359/154833280-2487eba0-f8c3-4520-9b93-eda727f828c5.png">

I decided to try and run the action from the Home Assistant UI using "Run Actions" and that doesn't do anything either.

View from HA
<img width="1049" alt="Screen Shot 2022-02-20 at 2 14 30 AM" src="https://user-images.githubusercontent.com/28831359/154832737-d9c5fa24-6ee8-4132-9e8b-5f48981db892.png">

. My HA logs don't show anything so I didn't know where to start this topic.

**Comment by EPMatt** [comment]
Hi @gac9415,

first of all I'd like to thank you for reaching out here and for your nice feedback. I'm happy that this blueprint is useful for you. :) 
Multiple conditions are checked before running the automation's action sequence; these include checks for debouncing, "ghost" or duplicate events etc. This means that, unfortunately, if the automation is not triggered by a device event as expected, the conditions won't be met and the action won't run. 

I'd suggest you to create a different automation with the same actions you provided to configure the automation with this blueprint, and use that in Node-Red.
Please let me know if you need any additional guidance, or if you have any other doubts. I'd be happy to help. 

Thank you for your patience! :)
